src/_guides/language/analysis-options.md

```diff
@@ -64,38 +88,43 @@ at the root of the package, in the same directory as the pubspec file.
   {% comment %}
   Tracking issue: https://github.com/dart-lang/sdk/issues/28385
   {% endcomment %}
-</aside>
+{{site.alert.end}}
 
 Here's a sample analysis options file:
 
-{% prettify yaml %}
+<?code-excerpt "analysis_options.yaml" from="include" remove="implicit-dynamic" retain="/^$|\w+:|- camel/" remove="http:"?>
+```yaml
```

Comment:
Interesting... the formatting for yaml got much less vibrant.

Reply:
Markdown code blocks are processed statically and rendered using the `highlight` package styles (as opposed to the prettifier JS package). I haven't yet tweaked `highlight` package styles in support of YAML. If you agree that this isn't gating, I'd suggest addressing it in a followup PR (though, if I have time before Filip or John review it, I ca add the style changes to this PR too).

Reply:
Definitely not gating. I'd noticed the difference in styles for awhile, but it's starting to annoy me more and more. :)

Reply:
Followup issue: #1918

Reply:
Addressed by #1924